index.html

```diff
@@ -3,9 +3,86 @@
+                  </div>
+            </section>
+            <hr>
+
+            <section class="mainProject">
+                <div>
+                <span class= "projectImgs"><img src="https://picsum.photos/250" alt="project image"></span>
+                
+                 <h3>Project 1</h3>
+                  <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nam delectus veniam, reprehenderit voluptatem molestiae enim obcaecati tempora, sunt consectetur necessitatibus repellat corporis? Labore debitis repudiandae aspernatur corrupti ea nobis quaerat!</p>
+              </div>
+
+              <div>
+                  <span class="projectImg2"><img src="https://picsum.photos/250" alt="project image"></span>
+                   <h3>Project 2</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quaerat praesentium tempora aliquam. Dignissimos, odio voluptatem, incidunt ipsam debitis rerum id dolorem, non architecto praesentium tempore reiciendis natus aliquam sed in?</p>
+              </div>
+
+              <div>
+                  <span class="projectImg3"><img src="https://picsum.photos/250" alt="project image"></span>
+                   <h3>Project 3</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quia animi, totam odio enim eveniet consectetur labore aperiam. Consectetur quo nam ratione eius accusantium cumque nobis esse consequatur ullam, quam omnis.</p>
+              </div>
+
+              <div>
+                  <span class="projectImg4"><img src ="https://picsum.photos/250" alt="project image"></span>
+                   <h3>Project 4</h3>
+                     <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Similique quod assumenda reprehenderit odit illum dolorum libero enim vero, veritatis ad, suscipit harum ab esse odio doloribus maxime distinctio beatae aliquid!</p>
+              </div>
+            </section>
+
+        </section>
+
+        <footer class="footer">
+            <h2>Like What You See?</h2>
+              <button>Contact Me</button>
+                <ul class="icons">
```

Comment:
awesome job adding in your social medias!! your entire structure looks GREAT! awesome work!